uaa_client/fake_uaa_provider.py

```diff
@@ -39,17 +43,25 @@ def authorize(request):
     return HttpResponseRedirect('%s?%s' % (url, qs))
 
 
+@uaa_refresh_exempt
```

Comment:
We need to exempt our fake UAA's token renewal endpoint, or else the app will get stuck in an infinite loop when token auto-renewal happens!